Resources/Prototypes/_Sunrise/Catalog/ert_uplink_catalog.yml

```
@@ -662,6 +662,18 @@
   - !type:ListingLimitedStockCondition
     stock: 2
 
+- type: listing
+  id: NTUplinkSunrisePouchExplosiveFilled
+  productEntity: PouchExplosiveFilled
+  icon: { sprite: _Sunrise/Interface/Misc/nt_icons.rsi, state: pouchgrenadefilled }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)